photoshell/photo.py

```diff
@@ -17,17 +17,29 @@
 ])
 
 
+photo_cache = {}
+
+
 class Photo(_Photo):
 
     # TODO: move this off the class
```

Comment:
The next branch I'm doing is going to replace those entirely, so I don't see a point in shuffling them around right now.